models/repo.go

```diff
@@ -353,6 +353,8 @@ func (repo *Repository) innerAPIFormat(e Engine, mode AccessMode, isParent bool)
 
 	repo.mustOwner(e)
 
+	numReleases, _ := GetReleaseCountByRepoID(repo.ID, FindReleasesOptions{IncludeDrafts: false, IncludeTags: true})
```

Comment:
same way as UI calc it

Reply:
Then should we remove repo.NumReleases ?

Reply:
@lunny your commend was hard to find :sweat_smile:  ...
I have a check how many files are afected.